docs/Tutorials/OpenAPI.md

```diff
@@ -304,6 +401,97 @@ the JSON response payload defined is as follows:
+
+```
+
+### Headers
+
+To define a reusable header definition you can use the [`Add-PodeOAComponentHeader`](../../Functions/OpenApi/Add-PodeOAComponentHeader) function. You'll need to supply a Name, and optionally any Content/Header schemas that define the responses payload.
+
+```powershell
+New-PodeOAIntProperty -Format Int32 -Description 'calls per hour allowed by the user'| Add-PodeOAComponentHeader -Name 'X-Rate-Limit'
+New-PodeOAStringProperty -Format Date-Time -Description 'date in UTC when token expires'| Add-PodeOAComponentHeader -Name 'X-Expires-After'
+
+ Add-PodeRoute -PassThru -Method Get -Path '/user/login' -ScriptBlock {
+              #code
+} | Set-PodeOARouteInfo -Summary 'Logs user into the system.' -Description 'Logs user into the system.' -Tags 'user' -OperationId 'loginUser' -PassThru |
+    Add-PodeOAResponse -StatusCode 200 -Description 'Successful operation' -Content (New-PodeOAContentMediaType -MediaType 'application/json', 'application/xml' -Content 'string' ) -Header @('X-Rate-Limit', 'X-Expires-After') -PassThru |
+    Add-PodeOAResponse -StatusCode 400 -Description 'Invalid username/password supplied'
+```
+
+
+### CallBacks
+
+To define a reusable callback definition you can use the [`Add-PodeOAComponentCallBack`](../../Functions/OpenApi/Add-PodeOAComponentCallBack) function. You'll need to supply a Name, and optionally any Content/Header schemas that define the responses payload.
+
+```powershell
+Add-PodeOAComponentCallBack -Name 'test' -Path '{$request.body#/id}' -Method Post  -RequestBody (New-PodeOARequestBody -Content @{'*/*' = (New-PodeOAStringProperty -Name 'id') } ) -Response (
+    New-PodeOAResponse -StatusCode 200 -Description 'Successful operation' -Content (New-PodeOAContentMediaType -MediaType 'application/json', 'application/xml' -Content 'Pet' -Array) |
+        New-PodeOAResponse -StatusCode 400 -Description 'Invalid ID supplied' |
+        New-PodeOAResponse -StatusCode 404 -Description 'Pet not found' |
+        New-PodeOAResponse -Default   -Description 'Something is wrong'
+)
+
+ Add-PodeRoute -PassThru -Method Post -Path '/petcallbackReference'  -Authentication 'Login-OAuth2' -Scope 'write'  -ScriptBlock {
+    #code
+} | Set-PodeOARouteInfo -Summary 'Add a new pet to the store' -Description 'Add a new pet to the store' -Tags 'pet' -OperationId 'petcallbackReference' -PassThru |
+    Set-PodeOARequest -RequestBody (New-PodeOARequestBody -Reference 'PetBodySchema' ) -PassThru |
+    Add-PodeOAResponse -StatusCode 200 -Description 'Successful operation' -Content (New-PodeOAContentMediaType -MediaType 'application/json', 'application/xml' -Content 'Pet' ) -PassThru |
+    Add-PodeOAResponse -StatusCode 405 -Description 'Validation exception' -Content @{
+       application/json' = ( New-PodeOAStringProperty -Name 'result' | New-PodeOAStringProperty -Name 'message' | New-PodeOAObjectProperty )
+    } -PassThru |
+    Add-PodeOACallBack -Name 'test1'   -Reference 'test'
+```
+
+
+### ResponseLink
```

Comment:
Tweak the value to `## Response Links`

Reply:
why? 
what is the difference between ResponseLink , CallBacks or Headers?

Reply:
It was just so it looked nicer, with the two words being separate. Though I have just spotted that I was meant to put `###` not `##`! Oops